client/blocks/importer/wordpress/import-everything/pre-migration/migration-ready.tsx

```diff
@@ -104,11 +105,37 @@ export function MigrationReady( props: Props ) {
 			) : (
 				<div className="import__pre-migration import__import-everything import__import-everything--redesign">
 					<div className="import__heading import__heading-center">
-						<Title>{ translate( 'You are ready to migrate' ) }</Title>
+						<Title>
+							{ translate( 'Your site is ready {{br/}} for its brand new home', {
+								components: {
+									br: <br />,
+								},
+							} ) }
+						</Title>
+					</div>
+					<div className="pre-migration__main-content">
+						<div className="pre-migration__main-content-primary-text">
+							{ translate(
+								"Enjoy a hassle-free migration process where we smoothly move your {{span}} website's content, media files, themes, plugins, and settings. {{/span}}",
+								{
+									components: {
+										span: <span />,
+										br: <br />,
+									},
+								}
```

Comment:
I don't see a `br` here! Also, do we need the whitespaces?

Reply:
Also I see we're using `br` in other texts, let's unify how we do it if possible

Reply:
Good catch. I removed the `span` and replaced it with `br`. We only need one space for mobile viewport when I hide the `br`, so that `your website` doesn't get squished together.